### ISSUE TITLE
Create multiple variants of thumbnails

### ISSUE THREAD

**Issue by stevenpetryk** [streamr-app/streamr-api#97]
Thumbnails should be created according to the multiple palettes, and then the correct thumbnail should be served based on the user's colorblindness preferences.
